contracts/contracts/utils/AerodromeAMOQuoter.sol

```diff
@@ -0,0 +1,544 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import { ICLPool } from "../interfaces/aerodrome/ICLPool.sol";
+import { IQuoterV2 } from "../interfaces/aerodrome/IQuoterV2.sol";
+import { IAMOStrategy } from "../interfaces/aerodrome/IAMOStrategy.sol";
+
+contract QuoterHelper {
```

Comment:
can you add a small comment here why the QuoterHelper is required. It might not be immediately obvious to others that try/catch in solidity requires a separate contract in order to function